### PR TITLE
I fixed the alignment bug

### PR DESCRIPTION
**Pull request by pabhi4881**
fixed the alignment bug of link icon 

issue no - #1024

### CODE REVIEW

**Review by swapnilsparsh — CHANGES_REQUESTED** [review]
Why did you changed the spelling? It is wrong.

**Comment by pabhi4881** [comment]
Okay, will decreasing the margin between the link icon and text be okay?

**Comment by pabhi4881** [comment]
@swapnilsparsh I have reduced the padding between the link icon and text.

File: index.html [hunk 1]
```diff
@@ -1420,11 +1420,11 @@ <h4>Whatsapp Message Sender</h4>
     </div>
     <div class="item">
         <img src="30DaysOfJavaScript/assets/154.png" alt="Word For Alphabet Speak Aloud">
-        <h4 style="font-size: 18px;">Word For Alphabet Speak Aloud</h4>
```
Comment (swapnilsparsh) [inline index.html]: Again you are making changes with font-size. Dont change font size

**Review by swapnilsparsh — CHANGES_REQUESTED** [review]

**Comment by pabhi4881** [comment]
@swapnilsparsh fixed everything

**Review by swapnilsparsh — CHANGES_REQUESTED** [review]
It is not fixed yet. Just by adding padding 0px won't fix this.

**Comment by pabhi4881** [comment]
What else do I need to do, sir? I have set the padding to 0px, which has resulted in the default padding being set to 0 and the link icon being aligned properly. If there is anything else I need to do, please suggest it to me, sir.

**Comment by swapnilsparsh** [comment]
@pabhi4881 "Movie TV Series Quote Generator" is not fixed yet.

**Comment by pabhi4881** [comment]
@swapnilsparsh fixed, Sir.

**Pull request merged**